src/interfaces/chain.cpp

```diff
@@ -301,10 +296,13 @@ class ChainImpl : public Chain
         auto it = ::mempool.GetIter(txid);
         return it && (*it)->GetCountWithDescendants() > 1;
     }
-    void relayTransaction(const uint256& txid) override
+    bool broadcastTransaction(const CTransactionRef& tx, uint256& txid, std::string& err_string, const CAmount& absurdfee, bool wait_callback) override
```

Comment:
This interface function doesn't need to expose the `wait_callback` bool since it's only ever called with `wait_callback` set to `false`.

Reply:
Ah yes, thanks done